_posts/2015-05-28-omero-bio-formats-5-1-2.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: Release of OMERO & Bio-Formats 5.1.2
+intro-blurb: The OME Consortium are pleased to announce the release of OMERO & Bio-Formats 5.1.2
+---
+Today we are releasing OMERO and Bio-Formats 5.1.2. This is a point release that contains bug-fixes, but also adds several new features to Bio-Formats and OMERO.
+
+**Bio-Formats**
+
+Bio-Formats 5.1.2 adds writing support for OME-TIFF to the native C++ implementation, allows export of OME BigTIFF, and adds a Slidebook6 reader (many thanks to Richard Myers of 3i - Intelligent Imaging Innovations (https://www.intelligent-imaging.com)
```

Comment:
url should be ``[https://www.intelligent-imaging.com](https://www.intelligent-imaging.com)``